### DISCUSSION TITLE
no longer access the ExplorerPatcher settings

### DISCUSSION THREAD

**Discussion by dtmaster79** [valinet/ExplorerPatcher#2085] [General]
After Windows Restart I can no longer access the ExplorerPatcher settings. Every time I right click on the taskbar it restarts. But I can only get into the settings by right-clicking. For updates etc. I find this to be a bit of a stupid solution.

**Comment by Poopooracoocoo** [comment]
Can you not access it from the Start Menu or Windows Search?

**Comment by pyrates999** [comment]
mentioned here: https://github.com/valinet/ExplorerPatcher/issues/2051

to run the explorer patcher properties application, run this from the run menu:
C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI

Possible fixes:

1. In ExplorerPatcher Properties, go to Advanced and tick **Do not change the taskbar context menu**
2. In ExplorerPatcher Properties, go to File Explorer and untick **Disable the windows 11 context menu**
3. In ExplorerPatcher Properties, go to Taskbar and set **Taskbar style** to **Windows 11**

**Reply by livejamie** [reply]
The run command worked for me but none of the 3 fixes worked for me.

**Reply by pyrates999** [reply]
You tried all 3 fixes at once?

**Reply by Vulosophy** [reply]
I used all of these, however, now I cannot access the settings using the taskbar or the win+x shortcut. When the win+x shortcut is enabled, I still do not see the option appear on the menu

**Reply by pyrates999** [reply]
Sorry, try this:

In ExplorerPatcher Properties, go to Advanced and untick **Do not change the taskbar context menu** - there should be an X beside it

You should then see **properties** when you right click on the taskbar.  However it was never added to the windows key + X menu.
